### DISCUSSION TITLE
TUI Design tools?

### DISCUSSION THREAD

**Discussion by laduke** [charmbracelet/bubbletea#220] [Q&A]
Hey, does anyone know/use a drawing app that works well for drawing terminal apps? 
I'm not that fast at bubbletea yet, so I get lazy on the design from the friction.

**Comment by muesli** [comment]
Back in my day we used `TheDraw` https://en.wikipedia.org/wiki/TheDraw for this 😆

**Reply by statik** [reply]
https://asciiflow.com/#/ may be useful.

**Comment by laduke** [comment]
https://monodraw.helftone.com/

**Comment by meowgorithm** [comment]
I'm not convinced terminal drawing tools will help too much with production-level TUI code, but they're definitely good for the concept phase. I'm personally a fan of [Tart](https://github.com/jtdaugherty/tart).
